src/libcmd/installables.cc

```diff
@@ -74,6 +74,16 @@ MixFlakeOptions::MixFlakeOptions()
         .handler = {&lockFlags.commitLockFile, true}
     });
 
+    addFlag({
```

Comment:
It might be nice to change this to a config setting, since then it can be specified on a per-project basis in `flake.nix`'s `nixConfig` attribute.